stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/call/RtcSession.kt

```diff
@@ -767,8 +785,11 @@ public class RtcSession internal constructor(
         logger.i { "[cleanup] #sfu; #track; no args" }
 
         coroutineScope.launch {
-            sfuConnectionModule.socketConnection.disconnect()
-            sfuConnectionMigrationModule?.socketConnection?.disconnect()
+            serialProcessor.submit("cleanupSfuConnections") {
```

Comment:
Tbh this code looks ugly to me _(The nesting of serialProcessor.submit() inside `coroutineScope.launch` )_ Maybe we can improve the code style. Wdyt?

```
coroutineScope.launch {
    serialProcessor.submit("..") {}
}
```

Reply:
Maybe can have an extension function